Add 11.31 release notes (#460)
mondoohq/docs: releases/2024-11-19-mondoo-11.31-is-out.md

---
slug: mondoo-11.31-is-out/
title: Mondoo 11.31 is out!
description: Announcing the 11.31 release of Mondoo with asset inventory exploration, NGINX security, expanded risk detection, and more!
authors: [tim]
image: /img/releases/2024-11-19-mondoo-11.31-is-out/query_pack_assets.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.31 is out! This release includes asset inventory exploration, NGINX security, expanded risk detection, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### Explore query packs results space-wide

> Understand the configuration of systems throughout your spaces with query packs. Query pack pages now expose results returned for each asset. Use Mondoo's two-dozen out of the box query packs to expose important configuration data such as running processes or available memory. If those query packs don't include quite what you need, write your own query packs to expose business specific asset configuration data throughout your spaces.

Reviewer: ```suggestion
Understand the configuration of systems throughout your spaces with query packs. Query pack pages now expose results returned for each asset. Use Mondoo's two dozen out-of-the-box query packs to expose important configuration data such as running processes or available memory. If those query packs don't include quite what you need, write your own query packs to expose business-specific asset configuration data throughout your spaces.
```

> ![Space wide asset queries](/img/releases/2024-11-19-mondoo-11.31-is-out/query_pack_assets.png)

Reviewer: ```suggestion
![Space-wide asset queries](/img/releases/2024-11-19-mondoo-11.31-is-out/query_pack_assets.png)
```

### New CIS NGINX benchmark policies

Keep critical web servers, proxy servers, and load balancers secure with new CIS NGINX benchmarks. These six policies include 91 checks in total designed to ensure that NGINX is not only installed securely but configured to protect sensitive data from prying eyes.

![NGINX policy page](/img/releases/2024-11-19-mondoo-11.31-is-out/nginx_policy.png)

> ### Run Mondoo in Azure CloudShell

Reviewer: ```suggestion
### Run Mondoo in Azure Cloud Shell
```

> Secure your Azure subscriptions without complex Entra ID and Azure configuration. Now Mondoo can install and run directly in Azure Cloud Shell: Simply paste the Workstation Quick Setup command in Azure Cloud Shell.

Author: ```suggestion
Secure your Azure subscriptions without complex Azure credential configuration. Now Mondoo can install and run directly in Azure Cloud Shell: Simply paste the Workstation Quick Setup command in Azure Cloud Shell.
```

> Secure your Azure subscriptions without setting up integrations or copying credentials to your local workstation. Now Mondoo can install and run directly in Azure CloudShell. Follow the Security Workstation settings to start scanning in just minutes.

Reviewer: ```suggestion
Secure your Azure subscriptions without setting up integrations or copying credentials to your local workstation. Now Mondoo can install and run directly in Azure Cloud Shell. Follow the Security Workstation settings to start scanning in just minutes.
```

> Secure your Azure subscriptions without setting up integrations or copying credentials to your local workstation. Now Mondoo can install and run directly in Azure Cloud Shell. Follow the Security Workstation settings to start scanning in just minutes.

Reviewer: ```suggestion
Secure your Azure subscriptions without complex Entra ID and Azure configuration. Now Mondoo can install and run directly in Azure Cloud Shell: Simply paste the Workstation Quick Setup command in Azure Cloud Shell.
```

Reviewer: I dunno if that's right.

![Azure CloudShell Installation](/img/releases/2024-11-19-mondoo-11.31-is-out/cloudshell_install.png)

## 🧹 IMPROVEMENTS

### Priority findings on the spaces page

> Quickly understand the spaces that need your attention the most with priority findings shown on the spaces page. Priority findings is the total of all critical and high findings on the space, helping you to better prioritize your effort.

Reviewer: ```suggestion
Quickly understand the spaces that need your attention the most with a count of priority findings shown on the spaces page. The number of priority findings is the total of all critical and high findings on the space. Mondoo highlights them to help you to better prioritize your effort.
```

![Spaces page tiles](/img/releases/2024-11-19-mondoo-11.31-is-out/priority_findings.png)

### Faster load times in compliance

> Track your space compliance quicker than ever, with up to 10x faster load times in the Compliance Frameworks page.

Reviewer: ```suggestion
Track your compliance quicker than ever, with up to 10x faster load times in the Compliance Frameworks page.
```

### Additional risk factors for FTP servers

Prioritize the risks that matter the most with new risk factors for assets running vsftp, Pure-FTPd, and ProFTPD FTP servers.

![New risk factors in Security Model configuration](/img/releases/2024-11-19-mondoo-11.31-is-out/risk_factors.png)

### Updated CIS Debian 11 Benchmark policy 2.0.0

With 665 updates including new and improved checks, descriptions, and remediation steps, this updated keeps your Debian 11 systems secure against the latest threats.

### Resource updates

#### gitlab.project

- New `emptyRepo` field
- New `groupRunnersEnabled` field
- New `jobsEnabled` field
- New `sharedRunnersEnabled` field

#### gitlab.project.approvalsetting

- New `selectiveCodeOwnerRemovals` field

#### gitlab.project.member

- New `state` field
- New `username` field

## 🐛 BUG FIXES AND UPDATES

> - Improve rendering of descriptions and remediation steps in Mondoo Linux Security policy.

Reviewer: ```suggestion
- Improve the rendering of descriptions and remediation steps in the Mondoo Linux Security policy.
```

- Move all service checks in the Mondoo Linux Security policy into a Sensitive Services chapter.

> - Expand the Mondoo Linux Security policy `Ensure FTP servers are stopped and not enabled` check to include Pure-FTPd, and ProFTPD.

Reviewer: ```suggestion
- Expand the Mondoo Linux Security policy `Ensure FTP servers are stopped and not enabled` check to include Pure-FTPd and ProFTPD.
```

> - Expand the Mondoo Linux Security policy `Ensure IMAP and POP3 servers are stopped and not enabled` check to include the Cyrus IMAP server.

Reviewer: ```suggestion
- Expand the Mondoo Linux Security policy's `Ensure IMAP and POP3 servers are stopped and not enabled` check to include the Cyrus IMAP server.
```

> - Expand the Mondoo Linux Security policy `Ensure DNS server is stopped and not enabled`, `Ensure NIS server is stopped and not enabled`, and `Ensure talk server is stopped and not enabled` checks to support Debian and Ubuntu based systems.

Reviewer: ```suggestion
- Expand the Mondoo Linux Security policy's `Ensure DNS server is stopped and not enabled`, `Ensure NIS server is stopped and not enabled`, and `Ensure talk server is stopped and not enabled` checks to support Debian- and Ubuntu-based systems.
```

- Add remediation steps for SLES/openSUSE to the Mondoo Linux Security policy.
- Fix the Mondoo Linux Security policy's `Ensure SSH Protocol is set to 2` check incorrectly running on Ubuntu 24.04 systems.
- Fix the Mondoo Linux Security policy's `Ensure mail transfer agent is configured for local-only mode` incorrectly identifying the IPv6 loopback address.

> - Ensure remediation steps in the Mondoo Linux Security policy include service restarts where necessary.

Reviewer: ```suggestion
- Ensure that the remediation steps in the Mondoo Linux Security policy include service restarts where necessary.
```

- Expand the Mondoo Linux Security policy's `Ensure IMAP and POP3 servers are stopped and not enabled` check to include the Cyrus IMAP server.
- Expand the Mondoo Linux Security policy's `Ensure DNS server is stopped and not enabled`, `Ensure NIS server is stopped and not enabled`, and `Ensure talk server is stopped and not enabled` checks to support Debian- and Ubuntu-based systems.
- Add remediation steps for SLES/openSUSE to the Mondoo Linux Security policy.
- Fix the Mondoo Linux Security policy's `Ensure SSH Protocol is set to 2` check incorrectly running on Ubuntu 24.04 systems.
- Fix the Mondoo Linux Security policy's `Ensure mail transfer agent is configured for local-only mode` incorrectly identifying the IPv6 loopback address.
- Ensure that the remediation steps in the Mondoo Linux Security policy include service restarts where necessary.
- Fix failing compliance reports in some spaces.
- Fix some AWS and Azure platforms not displaying correctly in the inventory overview.
- Fix the link to the CloudFormation template during AWS Serverless integration setup with non-us-east-1 regions.

> - Support searching for software, compliance framework, compliance controls in space search.

Reviewer: ```suggestion
- Add support for searching for software, compliance framework, and compliance controls in a space.
```
